src/renderer/components/ft-list-playlist/ft-list-playlist.js

```diff
@@ -50,6 +83,26 @@ export default Vue.extend({
     }
   },
   methods: {
+    handleExternalPlayer: function () {
+      this.openInExternalPlayer({
+        externalPlayer: this.externalPlayer,
+        externalPlayerExecutable: this.externalPlayerExecutable,
+        cmdArgs: this.externalPlayerCmdArguments,
+        externalPlayerIgnoreWarnings: this.externalPlayerIgnoreWarnings,
+        externalPlayerCustomArgs: this.externalPlayerCustomArgs,
+        strings: this.$t('Video.External Player'),
+
+        watchProgress: 0,
+        playbackRate: this.defaultPlayback,
+        videoId: null,
+        playlistId: this.playlistId,
+        playlistIndex: null,
+        playlistReverse: null,
+        playlistShuffle: null,
+        playlistLoop: null
+      })
+    },
+
```

Comment:
I was under the assumption that that wasn't possible, or unwanted, since I didn't see any other action in `utils.js` fetching settings from the store. Do you have an example I can use?

Reply:
nvm. found a way.